.eslintrc.js

```diff
@@ -140,8 +140,9 @@ module.exports = {
     'strict': 'error',
     'spaced-comment': ['error', 'always'],
     'no-new': 'error',
-    'no-unused-vars': ['error', { vars: 'all', args: 'all' }],
```

Comment:
can you add the `{ vars: 'all', args: 'all' }` options to the typescript rule. I forgot to add that